addons/mass_mailing/views/snippets_themes.xml

```diff
@@ -62,6 +62,8 @@
                     <t t-snippet="mass_mailing.s_masonry_block" t-thumbnail="/mass_mailing/static/src/img/snippets_thumbs/s_masonry_block.svg"/>
                     <t t-snippet="mass_mailing.s_media_list" t-thumbnail="/mass_mailing/static/src/img/snippets_thumbs/s_media_list.svg"/>
                     <t t-snippet="mass_mailing.s_showcase" t-thumbnail="/mass_mailing/static/src/img/snippets_thumbs/s_showcase.svg"/>
+                    <t t-snippet="mass_mailing.s_blockquote" t-thumbnail="/mass_mailing/static/src/img/snippets_thumbs/s_blockquote.svg"/>
```

Comment:
I don't think these snippets were in the wrong section. They are indeed inner content, like in the website builder. Maybe when this was reported it didn't work?

Reply:
I think when he made that spec the fair of using them as inner content was broken and that's why he got confused. I believe him and I talked about it since then, a long time ago though.

Reply:
To be honest, I don't understand what you mean so I'm not sure @Goaman will understand either... ^^'

Reply:
Indeed, I don't understand

Reply:
The category these snippets is currently in is for snippets that can be added _within other snippets_. At one point, this was broken for the two snippets you moved, meaning that it was not possible to drop them within other snippets. When lna saw this, rather than thinking that the feature was broken, he thought the snippets were miscategorized. The feature has meanwhile long been fixed so these snippets really do belong to this category.

What I'm saying is this spec is 2 months old and has become obsolete.

IIRC I had received two conflicting reports: 1) these inner snippets can't be dropped within other snippets, and 2) the snippets don't seem to be inner snippets so they should not be categorized as such. Back then, we chose to fix report 1 so report 2 became obsolete.

Is this clearer?

Reply:
It is, thank you ! :) @Goaman I therefore suggest to revert this change.

Reply:
Thank you Antoine, commit removed :+1: